lib/bolt/outputter/human.rb

```diff
@@ -269,78 +270,104 @@ def print_tasks(tasks, modulepath)
 
       # @param [Hash] task A hash representing the task
       def print_task_info(task)
-        # Building lots of strings...
-        pretty_params = +""
-        task_info = +""
+        params = []
+
         usage = if Bolt::Util.powershell?
                   +"Invoke-BoltTask -Name #{task.name} -Targets <targets>"
                 else
                   +"bolt task run #{task.name} --targets <targets>"
                 end
 
-        task.parameters&.each do |k, v|
-          pretty_params << "- #{k}: #{v['type'] || 'Any'}\n"
-          pretty_params << "    Default: #{v['default'].inspect}\n" if v.key?('default')
-          pretty_params << "    #{v['description']}\n" if v['description']
-          usage << if v['type'].start_with?("Optional")
-                     " [#{k}=<value>]"
-                   else
-                     " #{k}=<value>"
-                   end
-        end
+        module_path = task.files.first['path'].chomp("/tasks/#{task.files.first['name']}")
+        module_path = 'built-in module' if module_path.start_with?(Bolt::Config::Modulepath::MODULES_PATH)
 
         if task.supports_noop
           usage << Bolt::Util.powershell? ? '[-Noop]' : '[--noop]'
         end
 
-        task_info << "\n#{task.name}"
-        task_info << " - #{task.description}" if task.description
-        task_info << "\n\n"
-        task_info << "USAGE:\n#{usage}\n\n"
-        task_info << "PARAMETERS:\n#{pretty_params}\n" unless pretty_params.empty?
-        task_info << "MODULE:\n"
+        task.parameters&.each do |name, info|
+          type = info['type'] || 'Any'
 
-        path = task.files.first['path'].chomp("/tasks/#{task.files.first['name']}")
-        task_info << if path.start_with?(Bolt::Config::Modulepath::MODULES_PATH)
-                       "built-in module"
-                     else
-                       path
-                     end
-        @stream.puts(task_info)
+          description = +""
+          description << wrap(info['description']) if info['description']
+          description << "\n#{type}"
```

Comment:
It seems like the descriptions already have a newline, so this results in 2 newlines which makes the output pretty big. I think we don't need another newline here (or can do something like  chomp the description and then add the newline)

Reply:
Hm, I think my terminal may be adding extra space as well, since this looks more like 3 lines to me than 2...
```
  retry               The number of retries in case of network connectivity failures                                                         
                                                                                                                                             
                      Optional[Integer]                                                                                                      
                      Default: 5
```
![task_show_output](https://user-images.githubusercontent.com/5055819/110021135-3a253400-7cdf-11eb-861e-596ece6310a8.png)

Reply:
@lucywyman Updated to format a string and print to the stream once. Hopefully that fixes the spacing issues. 🤞